File: Seccion 11 - Funciones en C++/Seccion_11_Dart/04_ejercicio_02.dart

import 'dart:io';

double numero;

void pedirDatos() {
  stdout.writeln('Digite un número: ');
  numero = double.parse(stdin.readLineSync());
}

void al_cuadrado(double n) {
  double elevacion;

  elevacion = n * n;

  print('El número $n al cuadrado es: $elevacion');
}

void main() {
  pedirDatos();
  al_cuadrado(numero);
}

File: Seccion 11 - Funciones en C++/Seccion_11_Python/04_ejercicio_02.py

numero = 0

def pedir_datos():
    global numero
    numero = float(input('Digite un número: '))

def al_cuadrado(n):
    elevacion = n * n

    print('El número {} al cuadrado es: {}'.format(n, elevacion))

pedir_datos()
al_cuadrado(numero)